source/src/js/main-page.js

```diff
@@ -0,0 +1,45 @@
+class MainPage extends Page {
+  // @param {string} template
+  constructor(template) {
+    super('main', template);
+    this.template = Handlebars.compile(`
+        <div data-carousel class="list-of-places-carousel"></div>
+        <div data-list-of-places class="list-of-places"></div>
+    `);
+    super.render();
+    this.carousel = new Carousel(this.node.querySelector("[data-carousel]"), 1);
+    this.listOfPlaces = new ListOfPlaces(
+      this.node.querySelector("[data-list-of-places]")
+    );
+
+    this.fillCarousel();
+    this.fillListOfPlaces();
+  }
+
+  // Добавляет в div-блок с атрибутом carousel слайды достопримечательностей
+  fillCarousel() {
+    this.getPlaces().then((places) => {
+      for (const [_, place] of places.entries()) {
+        this.carousel.appendSlide(place);
+      }
```

Comment:
```suggestion
      places.forEach((place) => {
        this.carousel.appendSlide(place);
      });
```
а ещё лучше научить карусель принимать массив данных чтобы сделать
```suggestion
      this.carousel.appendSlides(places);
```